### DISCUSSION TITLE
Rad Pro: Extend conversion factor range

### DISCUSSION THREAD

**Discussion by boromix** [Gissio/radpro#82] [Ideas]
Hi Gissio and thanks for radpro 2.0rc4. I removed the internal tube and put on my FS-600 a female BNC so I can use more sensitive external probes, in particular the Russian pancake SI-8B whose conversion factor is between 430 - 450 CPM/µSv/h but currently the conversion factor in radpro stop at 400 CPM/µSv/h. 

Could you please extend  the conversion factor range to 450 CPM/µSv/h?  

Thanks and regards

**Comment by Gissio** [comment]
Great suggestion. Does a max of 800 cpm/µSv/h sound good?

**Reply by alfmck** [reply]
For my external sensitive tube I use 600 CPM/mkSv/h. To have more correlation with the data, supplied on https://remap.jrc.ec.europa.eu/Advanced.aspx I have to use 1000-1200 CPM/mkSv/h.

**Reply by Gissio** [reply]
So 1600 cpm/µSv/h it is!

**Reply by boromix** [reply]
Perfect, thank you!

**Comment by DG0MG** [comment]
At some point I will get to the point where I will need this change... :-)
